modules/Emails/include/ListView/ListViewDataEmailsSearchOnIMap.php

```diff
@@ -0,0 +1,316 @@
+<?php
```

Comment:
The name ListViewDataEmailsSearchOnIMap seems strange because IMAP is an acronym.